.eslintrc.js

```diff
@@ -60,6 +77,61 @@ module.exports = {
     '@typescript-eslint/no-var-requires': 'off',
     '@typescript-eslint/no-unused-vars': 'off',
     '@typescript-eslint/array-type': ['warn', { default: 'generic', readonly: 'generic' }],
+    '@typescript-eslint/member-ordering': [
+      'error',
```

Comment:
Could you please change all linting exceptions here to `warn` instead of `error`?